go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/mendersoftware/go-lib-micro
+
+go 1.14
+
+require (
+	github.com/ant0ine/go-json-rest v3.3.2+incompatible
+	github.com/mendersoftware/inventory v0.0.0-20200522065057-a25956b1ec7b
```

Comment:
what we currently use is `a1bca36b3cff2262d5bc9347a9af06b90ae55de5` (_Tue Dec 3 14:02:17 2019 +0100_) this is latest commit, so, maybe lets bind it to `2.0.0b1-build2` inventory git tag? just suggestions and nitpicking. I understand that this upgrade is intentional?

Reply:
>  maybe lets bind it to 2.0.0b1-build2 inventory git tag? 

This is actually the commit hash for that tag (unfortunately the tag is not recognized as a semver tag, hence the v0.0.0).